src/http.js

```diff
@@ -44,11 +44,21 @@ class Http {
         return;
       }
 
+      // set headers
       xhr.setRequestHeader('Authorization', publishableKey);
       xhr.setRequestHeader('Content-Type', 'application/json');
       xhr.setRequestHeader('X-Radar-Device-Type', 'Web');
       xhr.setRequestHeader('X-Radar-SDK-Version', SDK_VERSION);
 
+      // set custom headers if present
+      const customHeaders = Cookie.getCookie(Cookie.CUSTOM_HEADERS);
```

Comment:
does this need to be stored in cookies? or could it be passed through?

Reply:
hm passed through how? with each API request?

Reply:
yeah guess so, it's prob fine since the custom headers will be the same across requests